ext/pcre/config0.m4

```diff
@@ -11,21 +11,10 @@ PHP_ARG_WITH(pcre-jit,,[  --with-pcre-jit         Enable PCRE JIT functionality
 
   if test "$PHP_PCRE_REGEX" != "yes" && test "$PHP_PCRE_REGEX" != "no"; then
 
+    # FIXME: don't define "I'm not bundled" as only versions installed in /usr
     if test "$PHP_PCRE_REGEX" = "/usr"; then
-      if test -z "$PKG_CONFIG"; then
-        AC_PATH_PROG(PKG_CONFIG, pkg-config, no)
-      fi
-      if test -x "$PKG_CONFIG"; then
-        AC_MSG_CHECKING(for PCRE2 10.30 or greater)
-        if $PKG_CONFIG --atleast-version 10.30 libpcre2-8; then
-          PCRE2_VER=`$PKG_CONFIG --modversion libpcre2-8`
-          AC_MSG_RESULT($PCRE2_VER)
-        else
-          AC_MSG_ERROR(PCRE2 version 10.30 or later is required to compile php with PCRE2 support)
-        fi
-        PCRE2_LIB=`$PKG_CONFIG --libs libpcre2-8`
-        PCRE2_INC=`$PKG_CONFIG --cflags libpcre2-8`
-      fi
+      PKG_CHECK_MODULES([PCRE2], [libpcre2-8 >= 10.30], ,
+        AC_MSG_ERROR(PCRE2 version 10.30 or later is required to compile php with PCRE2 support))
     fi
 
     dnl PCRE2 in a non standard  prefix should still have its config tool.
```

Comment:
The `/usr` check above and the whole if below can be dropped, as that's also fallback code.

Reply:
So this would automatically try to find the system version, then fallback on bundled code only if the system one isn't found?

Reply:
There's two level of checks there, first for yes/no, then for `/usr`, so unless `--with-pcre-regex` is specified with a directory, it would still use the bundled library.

Reply:
And then it ignores the argument entirely? I'm thinking it's best to check for "external" here.

Or... yes would look for external libraries and fall back on the bundled one, external/internal could force one or the other. If we want to get fancy, that is.